Video-Transformers-Kotlin/app/src/main/java/com/tokbox/sample/basicvideochat/MainActivity.kt

```diff
@@ -251,7 +251,7 @@ class MainActivity : AppCompatActivity(), PermissionCallbacks {
     class logoWatermark(private val resources: Resources) : CustomVideoTransformer {
 
         // Get the image in bitmap format
-        private var image: Bitmap = BitmapFactory.decodeResource(resources, R.drawable.vonage_logo)
+        private final var image: Bitmap = BitmapFactory.decodeResource(resources, R.drawable.vonage_logo)
```

Comment:
I don't know much about Kotlin. What is the difference between `final var` and `val`?

Reply:
Me netiher. It seems val is enough

Reply:
OK, it was just curiosity. Thx!